src/misc/image.v1.yaml

```diff
@@ -8,7 +8,7 @@ allOf:
             title: IIIF Image
             type: string
             format: uri
-            pattern: ^https://
+            pattern: ^https://.+/[^/?#\[\]@]+$ # http://iiif.io/api/image/2.1/#uri-encoding-and-decoding, with support for percent encoding
```

Comment:
Should improve the possible prefix to have percent-encode special characters (but allow slashes).

Reply:
Seems `https://iiif.elifesciences.org/lax:09560/elife-09560-fig1-v1.tif/full/full/0/default.jpg`/`https://iiif.elifesciences.org/lax:09560/elife-09560-fig1-v1.tif` has a lot of slashes? What's the example that should match this?

Reply:
Yep. (This property would have been better as pointing to the `info.json` file, would have made it less ambigious. This is the base URI: `{scheme}://{server}{/prefix}/{identifier}`)